internal/dao/stubs/stub_errors.go

```diff
@@ -12,16 +12,16 @@ var ContextReadError = errors.New("missing variable in context")
 var ContextSecondInitializationError = errors.New("trying to initialize context twice, please avoid that")
 var RSAGenerationError = errors.New("rsa key generation failed")
 
-func NewRecordNotFoundError(ctx context.Context, stubName string) dao.NoRowsError {
+func NewRecordNotFoundError(ctx context.Context, stubName dao.NamedForError) dao.NoRowsError {
 	return dao.NoRowsError{
-		Message: fmt.Sprintf("%s DAO record does not exist", stubName),
+		Message: fmt.Sprintf("%s DAO record does not exist", stubName.NameForError()),
 		Context: ctx,
 	}
 }
 
-func NewCreateError(ctx context.Context, stubName string) dao.Error {
+func NewCreateError(ctx context.Context, stubName dao.NamedForError) dao.Error {
```

Comment:
Why did you decide to do that change? does that make sense to pass the type? why not pass directly the result NameForError()?

Reply:
It is bit cleaner as if we decide to change the NamedForError to something else and provide more context, it will be easier to change. But I did not decide to do that, I've just went with what we have in `dao/sqlx/sqlx_errors`